components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml

```diff
@@ -4269,6 +4269,10 @@ paths:
               schema:
                 type: string
           content:
+            application/json:
+              schema:
+                type: string
+                format: binary
```

Comment:
How can we send a json with binary format..

Reply:
Can't we use binary format for non-entity-body?